src/test/scala/com/gd/twitteranalytics/ReportProcessorTestSuite.scala

```diff
@@ -0,0 +1,16 @@
+package com.gd.twitteranalytics
+
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+import org.apache.spark.sql.functions.{current_date, lit}
+import org.scalatest.FreeSpec
+
+trait ReportProcessorTestSuite extends FreeSpec with DataFrameSuiteBase{
```

Comment:
is this trait supposed to be used only for active users report?

Reply:
Sorry, but this is an outdated file. It's been removed.